include/aws/http/http2_stream_manager.h

```diff
@@ -97,6 +97,26 @@ struct aws_http2_stream_manager_options {
     void *shutdown_complete_user_data;
     aws_http2_stream_manager_shutdown_complete_fn *shutdown_complete_callback;
 
+    /**
+     * Optional.
+     * When set, connection will be closed if 5xx response received from server.
+     */
+    bool close_connection_on_server_error;
+    /**
+     * Optional.
+     * The period for all the connections held by stream manager to send a PING in seconds.
+     * If you specify 0, manager will NOT send any PING.
+     * Note: if set, it must be large than the time of ping timeout setting.
+     */
+    size_t connection_ping_period_sec;
+    /**
+     * Optional.
+     * Network connection will be closed if a ping response is not received
+     * within this amount of time (nanoseconds).
+     * If you specify 0, a default value of 3 seconds is used.
```

Comment:
trivial: let's just say "a default value is used" instead of documenting 3.

If we document 3, that documentation is going to get copy/pasted into the CRTs, then someday we'll change the defaults and the CRT documentation will be wrong